### PR TITLE
[schema] Move the binder into its own file.

### PR DESCRIPTION
**Pull request by pgavlin**
Just what it says on the tin.

### CODE REVIEW

File: pkg/codegen/schema/bind.go [hunk 1]
```diff
@@ -0,0 +1,1321 @@
+package schema
```
Comment (mikhailshilkov) [inline pkg/codegen/schema/bind.go:1]: Don't we lint for copyright headers?

**Review by mikhailshilkov — APPROVED** [review]
LGTM, I assume it's all mechanical move of code

File: pkg/codegen/schema/bind.go (hunk 1 above)
Comment (pgavlin) [reply pkg/codegen/schema/bind.go:15]: I don't think so. Fixed, in any case.

**Comment by pgavlin** [comment]
Azure failure is unrelated.

**Pull request merged**